blog/2024-02-15-private-api-gateways.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Understanding Private API Gateways
+description: Exploring what private API gateways are and why they're used in the context of the serverless architecture landscape.
+authors:
+  - rybickic
+tags: [private api gateway, secure api gateway, serverless api gateway, private api gateway example, private api gateway tutorial, api gateway vpc, private api gateway terraform, application gateway, private endpoint, winglang, wing cloud]
+hide_table_of_contents: true
+---
+
+
+The landscape of serverless architectures is constantly evolving - and the patterns used by developers today range from [microservices](https://en.wikipedia.org/wiki/Microservices), to [hexagonal architectures](https://en.wikipedia.org/wiki/Hexagonal_architecture_(software)), to [multi-tier architectures](https://en.wikipedia.org/wiki/Multitier_architecture), to [event-based architectures](https://en.wikipedia.org/wiki/Event-driven_architecture) (EDAs). One of the most important challenges every development team considers when designing and building their system is [how well it secures their application](https://franklyspeaking.substack.com/p/frankly-speaking-52620-cloud-security).
+
+In this post, we’re going to deep dive on the private API gateway pattern: including what a private API gateway entails, its distinct advantages, and why you or your team might opt for this route over other services.
+
+Based on our team’s experience, we’ll be focusing on the capabilities available when building private API gateways on AWS using their managed [VPC](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html) and [API Gateway](https://docs.aws.amazon.com/apigateway/latest/developerguide/welcome.html) services. But many of the lessons will also apply to cloud applications built using Azure’s [API Management service](https://learn.microsoft.com/en-us/azure/api-management/private-endpoint) and other major cloud providers. 
+
+## Understanding Private API Gateways
+
+If we simplify what a private API gateway is, it's a secure means of exposing a set of APIs within a private network, typically established using a [Virtual Private Cloud](https://en.wikipedia.org/wiki/Virtual_private_cloud) (VPC). Let’s first understand each of these.
```

Comment:
If we simply explain/describe?

Reply:
Not sure I understand the comment - happy to reword the sentence / paragraph if you have suggestions